src/main/core/main.c

```diff
@@ -250,6 +253,8 @@ gint main_runShadow(gint argc, gchar* argv[]) {
         }
     }
 
+    atexit(print_durations);
```

Comment:
Instead of logging to stderr atexit, can we log using Shadow's logger (i.e., `message()`) after `_main_helper()` returns (but before we shut down the logger) to make sure the information is stored more persistently?